src/features/formWizard/formWizard.tsx

```diff
@@ -0,0 +1,161 @@
+              {({ input, meta: { error, touched } }) => (
+                <FormControl isInvalid={error && touched}>
+                  <Input {...input} type="password" placeholder="Password" />
+                  <FormErrorMessage>{error}</FormErrorMessage>
+                </FormControl>
+              )}
+            </Field>
+          </Stack>
+        </WizardPage>
+        <WizardPage>
+          <Stack spacing={4}>
+            <Field name="phone" validate={zod2FieldValidator(phoneSchema)}>
+              {({ input, meta: { error, touched } }) => (
+                <FormControl isInvalid={error && touched}>
+                  <Input {...input} type="text" placeholder="Phone" />
+                  <FormErrorMessage>{error}</FormErrorMessage>
+                </FormControl>
+              )}
+            </Field>
+            <Field
+              name="billingAddress"
+              validate={zod2FieldValidator(required)}
+            >
+              {({ input, meta: { error, touched } }) => (
+                <FormControl isInvalid={error && touched}>
+                  <Input {...input} type="text" placeholder="Billing Address" />
+                  <FormErrorMessage>{error}</FormErrorMessage>
+                </FormControl>
+              )}
+            </Field>
+            <Field
+              name="shippingAddress"
+              validate={zod2FieldValidator(required)}
+            >
+              {({ input, meta: { error, touched } }) => (
+                <FormControl isInvalid={error && touched}>
+                  <Input
+                    {...input}
+                    type="text"
+                    placeholder="Shipping Address"
+                  />
+                  <FormErrorMessage>{error}</FormErrorMessage>
+                </FormControl>
+              )}
+            </Field>
+          </Stack>
+        </WizardPage>
+        <WizardPage>
+          <Stack spacing={4}>
+            <Field name="newsletter" type="checkbox">
+              {({ input, meta }) => (
+                <Checkbox
+                  {...input}
+                  size="lg"
+                  isChecked={input.checked}
+                  onChange={input.onChange}
+                >
+                  Subscribe to Newsletter
+                </Checkbox>
+              )}
+            </Field>
+            <Field name="specialOffers" type="checkbox">
+              {({ input, meta }) => (
+                <Checkbox
+                  {...input}
+                  size="lg"
+                  isChecked={input.checked}
+                  onChange={input.onChange}
+                >
+                  Receive Special Offers via Email
+                </Checkbox>
+              )}
+            </Field>
+            <Field name="smsNotifications" type="checkbox">
+              {({ input, meta }) => (
+                <Checkbox
+                  {...input}
+                  size="lg"
+                  isChecked={input.checked}
```

Comment:
this line is strange.

also this Checkbok component is reused in the same way many times.
looks to me the need to add a CheckbokFF adapter component between chakra and FF 

<CheckBoxField name=".." size="lg" > 
   Text 
</>

You can propagate all chakra style props.

in my original template there was an initial tentative to this.

Reply:
yes, i could not wait for everyone to review it so i merge the  react-router branch in the form-wizard feature when you approved it.
Seems reasonable to have a checkbox component for the forms since chackra checkbox props have not the same name of the standard one.

Reply:
Seems reasonable to have a checkbox component for the forms since chackra checkbox props have not the same name of the standard one.